test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs

```diff
@@ -48,6 +48,41 @@ public async Task CheckNonOptionalSettingsThrowsOnMissingAzureWebJobsStorage()
                 $"This is required for all triggers other than {string.Join(", ", Constants.TriggersWithoutStorage)}.");
         }
 
+        [Fact]
+        public async Task CheckNonOptionalSettingsDoesntThrowMissingStorageUsingManagedIdentity()
+        {
+
```

Comment:
nit -- remove this blank line. Some of our repos will log a build warning for this -- but this one must not.